internal/services/cdn/cdn_frontdoor_custom_domain_resource.go

```diff
@@ -127,30 +116,22 @@ func resourceCdnFrontDoorCustomDomain() *pluginsdk.Resource {
 			},
 		},
 	}
+
+	return resource
 }
 
 func resourceCdnFrontDoorCustomDomainCreate(d *pluginsdk.ResourceData, meta interface{}) error {
 	client := meta.(*clients.Client).Cdn.FrontDoorCustomDomainsClient
 	ctx, cancel := timeouts.ForCreate(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	profileId, err := parse.FrontDoorProfileID(d.Get("cdn_frontdoor_profile_id").(string))
+	profileId, err := parse.FrontDoorProfileIDInsensitively(d.Get("cdn_frontdoor_profile_id").(string))
```

Comment:
this is being validated above and so should be being parsed case-sensitively:

```suggestion
	profileId, err := parse.FrontDoorProfileID(d.Get("cdn_frontdoor_profile_id").(string))
```